### ISSUE TITLE
Styling lost with Jupyter-book using autoenum

### ISSUE THREAD

**Issue by StigKorsnes** [domdfcoding/enum_tools#52]
Hi!
Writing some documentation using jupyter-book, I decided to use this package for better looking api documention. (using sphinx ext.autodoc). However, some styling was lost when using autoenum. The same applies for other packages like sphinx-toolbox (providing other types like namedtuples), and pydantic models with autodoc_pydantic. However, authors of autodoc_pydantic was able to fix this by using sphinx events to add missing css classes. I.e "py enum" does not render ok, but "py class enum" does.

Please see [JB](https://github.com/executablebooks/jupyter-book/issues/1563) for the issue, and [autodoc_pydantic](https://github.com/mansenfranzen/autodoc_pydantic/issues/77) . Would you be willing to implement such a fix? Currently I do a post documentation build step, where I add class to all html elements with "py enum".

**Comment by domdfcoding** [comment]
Hi @StigKorsnes, sorry for the wait. I've made the necessary fix in version 0.9.0

Let me know if you find any issues.

**Comment by StigKorsnes** [comment]
@domdfcoding Thanks, works fine!
Did you also fix str enum ? I seem to recall not having the value last I tried it. I had an enum with members like  this:
```
Bpos="B+"
```
that previously rendered to 
```
Bpos= <RatingScaleEnum.Bpos:>
```
but now correctly renders to:
```
Bpos= <RatingScaleEnum.Bpos: 'B+'>
```
or is my memory playing tricks?
